matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/DefaultCryptoService.kt

```diff
@@ -549,21 +444,12 @@ internal class DefaultCryptoService @Inject constructor(
             return false
         }
 
-        val encryptingClass = MXCryptoAlgorithms.hasEncryptorClassForAlgorithm(algorithm)
-
-        if (!encryptingClass) {
+        if (algorithm != MXCRYPTO_ALGORITHM_MEGOLM) {
```

Comment:
OLM encrypted room not supported anymore? (not sure if current SDK handles it correctly though, web does)

Reply:
That's correct, is that even used anywhere anymore? The spec does mention this for example:

> In order to enable end-to-end encryption in a room, clients can send an m.room.encryption state event specifying m.megolm.v1.aes-sha2 as its algorithm property.

And nowhere mentions `m.olm.*` inside rooms, though I guess it doesn't really forbid it either.